fuzzing/Makefile.fuzzing_common

```diff
@@ -14,6 +14,7 @@ CFLAGS += -ggdb                # Make ASAN output more useful error messages
 CFLAGS += -D_FORTIFY_SOURCE=2  # Compiler hardening
 
 # Various utilitiy modules
+USEMODULE += gnrc_ipv6
```

Comment:
This change was done due to gnrc_ipv6 being mandatory (for all harness types) at the moment because `sys/fuzzing/` needs refactoring. :)